docs/pages/push-notifications/what-you-need-to-know.mdx

```diff
@@ -4,29 +4,29 @@ sidebar_title: About notification types
 description: Learn about notification types and their behavior before you get started.
 ---
 
-Notifications have a large surface area and differences across platforms can make implementing notifications intimidating.
+Notifications are alerts that inform users of new information or events, even when the app isn't actively in use. Notifications have a large surface area and differences across platforms can make implementing notifications intimidating.
 
-Whether you are starting with notifications or have existing knowledge, this document aims to give you a quick start by explaining the different notification types and their behavior that you should understand.
+Whether you are just starting with notifications or have existing knowledge, this document explains the different types of notification and their behaviors.
 
-Remember that Expo's notification support builds on top of the native functionality provided by Android and iOS. The same concepts and behaviors from native platforms apply to Expo apps. If you are unsure about a specific notification feature, see each platform's [official documentation](#external-references).
+Expo's notification support builds on top of the native functionality provided by Android and iOS. The same concepts and behaviors from native platforms apply to Expo apps. If you are unsure about a specific notification feature, see each platform's [official documentation](#external-references).
 
-## Remote and Local notifications
+## Remote and Local Notifications
```

Comment:
All the section titles follow the sentence case. Please revert them in orginial state. For more info, check: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings.

Reply:
Then should it not be "Remote and local notifications" ?  What's the rule about **Local** being capitalized but **notifications** not?  This seems inconsistent to me.  And could it not be argued that **Notifications** is the name of a product (or at least a feature)?

I defer to your call.  Just pointing out a confusion as I see it.